go-controller/go.mod

```diff
@@ -70,5 +70,4 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.20.0
 	k8s.io/mount-utils => k8s.io/mount-utils v0.20.0
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.0
-
```

Comment:
this commit doesn't look like its doing anything anymore? should we remove it?

Reply:
good catch, i see sriovnet was version bumped in https://github.com/ovn-org/ovn-kubernetes/commit/bfe617c45318a6a0b8d80bc3ae6823cf63c1585d

will drop this commit